app/res/xml/preferences_developer.xml

```diff
@@ -6,8 +6,8 @@
         android:key="cc-superuser-enabled"
         android:title="Developer Mode Enabled"/>
     <ListPreference
```

Comment:
Alright, though for future purposes would you be just able to confirm it by putting the enabled attr above defaultValue and formatting the xml file and see whether AS rearranges the enables after defaultValue. This style file  https://github.com/dimagi/commcare-android/blob/master/.android_studio_settings/codestyles/CommCare%20Coding%20Style.xml#L177 seems to indicate ordering as well but I am not really sure.